src/Files.App/ServicesImplementation/Settings/FoldersSettingsService.cs

```diff
@@ -274,6 +274,12 @@ public bool ShowFileExtensionWarning
 			set => Set(value);
 		}
 
+		public bool ShowCheckboxesInDetailsView
```

Comment:
```suggestion
		public bool ShowCheckboxesInDetailsLayout
```
We usually refer to it as the details layout.